packages/engine/src/simulation/comms/mod.rs

```diff
@@ -119,16 +119,18 @@ impl Comms {
         Ok(())
     }
 
+    /// TODO: docstring
```

Comment:
Could be DOC for easier search afterwards

Reply:
We should keep at least "TODO". "TODO DOC:" should work but I think "TODO: docstring" is fine, too.

Reply:
```suggestion
    /// TODO: DOC
```